.github/CONTRIBUTING.md

```diff
@@ -1,128 +1,87 @@
-  `01`, `02`, etc. In each chapter we restart the recipe counter.
-- Each recipe can have more than one example subfolder. These subfolders are
-  named `*example*`. Any code must reside in these subfolders.
-
-
-### Updating the table of contents and generate README files
-
-The README files that form the table of contents (main README.md, chapter READMEs and recipe READMEs
-are generated from title.txt and abstract.md files.
-
-This means that you should not modify README.md files but rather only edit title.txt and abstract.md files.
-
-To update the README files, run `python contributing/generate-readmes.py` - this file updates READMEs in place.
-
-
-### Coding style
-
-#### Indentation
-
-We use 2 spaces instead of 4 spaces to reduce the printed page width.
-No tabs.
-
-
-#### Case of commands
-
-We use lowercase for commands, i.e.:
-```cmake
-cmake_minimum_required(VERSION 3.5 FATAL_ERROR)
-```
-and not:
-```cmake
-CMAKE_MINIMUM_REQUIRED(VERSION 3.5 FATAL_ERROR)
-```
-
-
-#### Line continuation
-
-It is OK to put commands on one line if it improves readability, e.g.:
-```cmake
-list(APPEND CXX_BASIC_FLAGS "-g3" "-O1")
-```
-
-For line continuation we use the following style:
-```cmake
-target_compile_options(asan-example
-  PUBLIC
-    ${CXX_BASIC_FLAGS}
-    ${_asan_flags}
-  )
-
-set(_whathaveyou
-  item1
-  item2
-  item3
-  )
-```
-
-
-#### Variable names
+  integration services return passing checkmarks, and maintainers give "Approved" reviews.
 
-Start an internal variable (i.e. one that is not exposed to the user) with an
-underscore:
-```cmake
-set(PUBLIC_VARIABLE "this one is exposed")
+## Licensing
 
-set(_temp "this one is internal")
-```
+* We do not require any formal copyright assignment or contributor license agreement.
+* **Any contributions intentionally sent upstream are presumed to be offered under the terms of the OSI-approved [MIT License].**
+* You should read the [DCO] and certify that you agree to it by **signing off** your patch using the [`git commit -s` command].
+  Note that signing off and GPG-signing a commit are not the same thing and **we do not require** [GPG-signing] your commits.
+  We use a [DCO bot] to ensure that submitted patches are signed off. The bot
+  will block merging and suggest actions to resolve the block.
 
 # Additional Resources
 
 * [General GitHub documentation](https://help.github.com/)
 * [PR best practices](http://codeinthehole.com/writing/pull-requests-and-other-good-practices-for-teams-using-github/)
 * [A guide to contributing to software packages](http://www.contribution-guide.org)
 * [Thinkful PR example](http://www.thinkful.com/learn/github-pull-request-tutorial/#Time-to-Submit-Your-First-PR)
+* [Developer Certificate of Origin versus Contributor License Agreements](https://julien.ponge.org/blog/developer-certificate-of-origin-versus-contributor-license-agreements/)
+
+
+[Psi4 contribution guide]: https://github.com/psi4/psi4/blob/master/.github/CONTRIBUTING.md
```

Comment:
Since you quote Psi4 again here this must be taken as an example or is there a relationship between psi4 and dev- afe I ma not aware of?

Reply:
This one was a leftover from a copy-paste.